templates/block--lb-node-locations.html.twig

```diff
@@ -0,0 +1,11 @@
+<div{{ attributes }}>
```

Comment:
Let's always provide docblock for twig templates with all variables available in template context. See core node.twig as an example